backend/pkg/server/restaurant.go

```diff
@@ -29,3 +29,15 @@ func (h *restaurantHandler) exists(w http.ResponseWriter, r *http.Request) {
 
 	respondJSON(w, http.StatusOK, exists)
 }
+
+func (h *restaurantHandler) listRestaurants(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	listRestaurants, err := h.restaurantService.ListRestaurants(ctx)
```

Comment:
ta zmienna to raczej `restaurantsList`, czyli lista restauracji, zwracana przez handler `listRestaurants` - wylistuj restauracje